pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
Looks like this should actually be `x - y`, would you mind correcting this please?

Reply:
Sorry about the mistake there, I've now corrected it and the tests pass. I'll be more careful in the future.